cg/cli/demultiplex/demux.py

```diff
@@ -167,3 +170,18 @@ def delete_flow_cell(
             run_dir=run_dir,
             status_db=status_db,
         )
+
+
+@click.command(name="copy-completed")
+@click.pass_obj
+def copy_novaseqx_flow_cells(context: CGConfig):
+    """Copy novaseqx flow cells ready for post processing to demultiplexed runs."""
+    flow_cells = Path(context.demultiplex.out_dir)
+    demultiplexed_runs = Path(context.demultiplex.run_dir)

```

Comment:
Can you double-check this? I think the names are switched: `run_dir` is for the flow cell runs and `out_dir` for demultiplexed runs

Reply:
Good catch, thanks!